### DISCUSSION TITLE
DHCP problem?

### DISCUSSION THREAD

**Discussion by canerpense** [DesktopECHO/Pi-hole-for-Android#72] [General]
Even if the DHCP server option is enabled in pi-hole, and all DHCP related ports are open in the network, somehow the container blocks DHCP ports, in my case. As a result DHCP server is not advertised in the network and no device connects to DHCP.

**Comment by DesktopECHO** [comment]
Check to make sure the DHCP service is running, what is the output of:

`android@channel:~$ sudo ss -lpun 'sport = :67'`

You should see `pihole-FTL`:

```
State                 Recv-Q                Send-Q                               Local Address:Port                                 Peer Address:Port                Process                
UNCONN                0                     0                                          0.0.0.0:67                                        0.0.0.0:*                    users:(("pihole-FTL",pid=13310,fd=4))
```
You can also try flushing the firewall rules to see if one is in the way:

`sudo iptables-legacy --flush`
